tap/tcp_assembler.go

```diff
@@ -89,7 +89,9 @@ func (a *tcpAssembler) processPackets(dumpPacket bool, packets <-chan source.Tcp
 			}
 			diagnose.InternalStats.Totalsz += len(tcp.Payload)
 			a.assemblerMutex.Lock()
-			a.AssembleWithContext(packet.NetworkLayer().NetworkFlow(), tcp, &c)
+			if os.Getenv("MIZU_TAPPER_NO_ASSEMBLER") != "true" {
+				a.AssembleWithContext(packet.NetworkLayer().NetworkFlow(), tcp, &c)
+			}
```

Comment:
```suggestion
			if os.Getenv("MIZU_TAPPER_DISABLE_TCP_REASSEMBLY") != "true" {
			        a.assemblerMutex.Lock()
				a.AssembleWithContext(packet.NetworkLayer().NetworkFlow(), tcp, &c)
				a.assemblerMutex.Unlock()
			}
```

Reply:
@dvdlevanon, can we skip `Lock`/`Unlock` if `MIZU_TAPPER_NO_ASSEMBLER == "true"`?

Reply:
yes